char/liberate/SCRIPTS/sedFullModels

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
can you add some description about what that script is for as a comment? is it needed to be run everytime you want to build the standard cell, or just for initially importing the repo structure?